CHANGELOG.md

```diff
@@ -113,9 +114,9 @@ There isn't a valid core binary for this release. Use v0.57.2 instead.
 
 ### 💡 Enhancements 💡
 
-- Make the in-memory and persistent queues more consistent (#5764)ś
+- Make the in-memory and persistent queues more consistent. (#5764)
 - `ocb` now exits with an error if it fails to load the build configuration. (#5731)
-- Deprecate `HTTPClientSettings.ToClientWithHost` (#5737)
+- Deprecate `HTTPClientSettings.ToClientWithHost`. (#5737)
```

Comment:
Confused about these changes...

Reply:
Just making the formatting consistent

Reply:
Please split them in a separate PR :)

Reply:
reverted and rebased

Reply:
I can rebase this again. Is it ready to merge when I do?

Reply:
@bogdandrutu, I believe this PR is pending on your side. Would you be able to review/merge this?

Reply:
I think it should be your side as well, so we should not have "refactoring, formatting" changes in the same PR. I sent a separate PR https://github.com/open-telemetry/opentelemetry-collector/pull/6012